content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -25,31 +25,36 @@ is a replacement for the `istio-init` container that performs the same
 networking functionality but without requiring Istio users to enable elevated
 Kubernetes RBAC permissions.
 
-The Istio CNI plugin performs the Istio mesh pod traffic redirection in the Kubernetes pod lifecycle's network
+The Istio CNI plugin identifies Istio user application pods with Istio sidecars requiring traffic redirection and
+performs the Istio mesh pod traffic redirection in the Kubernetes pod lifecycle's network
 setup phase, thereby removing the [requirement for the `NET_ADMIN` and `NET_RAW` capabilities](/docs/ops/deployment/requirements/)
 for users deploying pods into the Istio mesh.  The Istio CNI plugin
 replaces the functionality provided by the `istio-init` container.
```

Comment:
Could maybe use a `{{note}}` that this is a cni *plugin* - it is not a full on CNI, and doesn't replace those.

Reply:
We had a section about Istio CNI needs to work with other main plugins. I added a tip here to make it more obvious and point to that section.